### ISSUE TITLE
Loading Screen - SpriteKit Spin Wheel

### ISSUE THREAD

**Issue by ToreaBayBlue** [hsilived/SpinWheel#1]
I'm looking for way to make the startup and reset more fluid. Right now the wheel defaults to the "GameScene.sks" until you press the button to continue, it's almost like a loading screen with the "SpriteKit Spin Wheel" text. 

Is there a way to remove this? To where it will just load and display the functioning spin wheel.

**Comment by hsilived** [comment]
This isn't an issue, it's a feature request nor is this the place for feature requests. You are free to fork this project and adjust how you see fit, as I've released to the public with an open license. 

Just so you are aware I spent days perfecting the physics, feel and graphics for this project. You didn't even say anything nice about it or "star" it before asking for changes...lol. 

Best of luck with your spin wheel project.

**Comment by ToreaBayBlue** [comment]
Really like the project, thumbs up & starred.

I was able to remove the initial loading/landing/setup screen in the "GameScene.swift" class by running the displaySpinWheel function like so. 

class GameScene: SKScene, SKPhysicsContactDelegate {
    private var spinWheelOpen = false
    private var spinWheel: SpinWheel!
    override func didMove(to view: SKView) {  
       self.physicsWorld.contactDelegate = self

        _displaySpinWheel()_
    }

Removed the loading/landing/setup screen after the spin by setting backgroundBlocker as a global variable in SpinWheel class. 

var backgroundBlocker: SKSpriteNode = SKSpriteNode()

then updated the closeSpinWheel function like so. 

     backgroundBlocker.removeFromParent()
